examples/elasticsearch/src/main/kotlin/cnt/elasticsearch/BookService.kt

```diff
@@ -0,0 +1,153 @@
+package cnt.elasticsearch
+
+import cnt.elasticsearch.ElasticSearchConsts.SEARCH_API
+import io.vertx.core.json.JsonArray
+import io.vertx.core.json.JsonObject
+import org.apache.http.util.EntityUtils
+import org.elasticsearch.client.Request
+import org.elasticsearch.client.Response
+import org.elasticsearch.client.RestClient
+import javax.enterprise.context.ApplicationScoped
+
+
+/**
+ * This Service provides access to the elastic search cluster
+ */
+@ApplicationScoped
+class BookService(
+    private val elasticSearchClient: RestClient,
+) {
```

Comment:
I think this would be more of a `BookRepository`, at least as far as stereotypes are concerned.

Reply:
True 🤝